pkgs/development/tools/kafka-delta-ingest/default.nix

```diff
@@ -33,11 +40,12 @@ rustPlatform.buildRustPackage rec {
 
   cargoLock = {
     lockFile = ./Cargo.lock;
-    outputHashes = {
-      "deltalake-0.4.1" = "sha256-0v3n+qMbBhw53qPuZdhGSO+aqc6j8T577fnyEIQmPDU=";
-    };
```

Comment:
Since no git crates are left you should switch to `fetchCargoTarball`.

Reply:
I'll see if I can make another pr soon